euth_wagtail/settings/base.py

```diff
@@ -371,6 +371,7 @@
 FLASHPOLL_BACK_URL = "http://int.backend.flashpoll.eu/flashpoll/api/v7/management"
 FLASHPOLL_BACK_USER = "fp_management"
 FLASHPOLL_BACK_PASSWORD = "53c4100e8ab143fe59fcb2e743cf4aba662ad25lacab0eb37fb9c69d8f27363fa19f0531bd681"
+GOOGLE_API_KEY = "AIzaSyC8kq3VbEzLA1xqe0ItRk-y4bgAg89h4Qc"
```

Comment:
If you add this to your local settings/local.py it won't be visible to everyone on Github